.gitignore

```diff
@@ -69,3 +69,6 @@ hw/top_englishbreakfast/ip/xbar_peri/xbar_peri.core
 # Rust Cargo build system files.
 Cargo.lock
 sw/host/**/target
+
+# pip source folder
+src
```

Comment:
Remove this exclude please, it's way too generic (and unnecessary once we fix the editable install, see below).

Reply:
Done.